ui/partials/perf/alertsctrl.html

```diff
@@ -221,9 +221,11 @@
   </p>
 </div>
 
-<div class="well" ng-show="getMoreAlertSummariesHref">
-    <div class="btn btn-default btn-sm"
-         ng-click="getMoreAlertSummaries(count)">
-      Get more...
-  </div>
+<div class="well">
```

Comment:
Let's change the class here from `well` to `text-center`. That will center the alerts, which looks a little nicer. :)